codejar.ts

```diff
@@ -440,3 +440,15 @@ export function CodeJar(editor: HTMLElement, highlight: (e: HTMLElement) => void
     },
   }
 }
+
+
+// Allows the use of `new CodeJar()` in TypeScript without raising the
+// "TS7009: 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type"
+// warning.
+export declare interface CodeJarConstructor {
```

Comment:
This approach is really messed. I fought it will be easier and won't require `any` casting. 

Well, maybe it's time to introduce BC? And drop `new` keyword. Also, release version under v3.

Reply:
haha yeah it's pretty gnarly. If there is a prettier way to do it, it is beyond my TS skill level. I'll leave the decision to you about the BC. 

If you *do* opt for BC and v3, I'd also suggest to lowercase the `CodeJar` function, since convention suggests that it should be constructible when it starts with uppercase.